weaviate/collections/classes/filters.py

```diff
@@ -541,7 +541,7 @@ class _MultiTargetRef(_WeaviateInput):
 
 
 _TargetRefs = Union[_SingleTargetRef, _MultiTargetRef]
-_FilterTargets = Union[_SingleTargetRef, _MultiTargetRef, str]
+_FilterTargets = Union[_SingleTargetRef, _MultiTargetRef, str, None]
```

Comment:
Instead of using None, should we have a special extra field for reference counts? That would require weaviate changes, but might make thigns more explicit 🤔

Reply:
I agree that explicit would be better. I didn't have a good idea at the time for it but looking at it again now I see a nice path so will implement it server-side. I will also use the PR to look at the surprising nil pointer deref in the tests